### PR TITLE
GHL Methods

### PR DESCRIPTION
**Pull request by raminka13**
- [x] to_ghl methods in each model.

### CODE REVIEW

File: app/models/discover_flight.rb
Comment (timjedrek) [inline app/models/discover_flight.rb:1]: Need to add the note payload and map it in GHL.
===========================================
          Select aircraft category:  #{self.aircraft}
    
          Is there anything else you'd like us to know: #{self.comments}
        
          Selected Date: #{self.preferred_date}
          Selected Time: #{self.preferred_availability}
          Alternate Date: #{self.alternate_date}
          Alternate Time: #{self.alternate_availability}

File: app/models/enrollment.rb
Comment (timjedrek) [inline app/models/enrollment.rb:1]: Same with this.. no note payload.

File: app/models/quiz_result.rb
Comment (timjedrek) [inline app/models/quiz_result.rb:1]: Same with this, no note payload.

File: app/models/recreational_flight.rb
Comment (timjedrek) [inline app/models/recreational_flight.rb:1]: Same with this.. no note payload.

**Review by timjedrek — CHANGES_REQUESTED** [review]

**Comment by raminka13** [comment]
- [x] Added the remaining fields for each form in their respective model method.

File: app/models/checkout.rb
[the remark this answers is not in the document]
Comment (timjedrek) [reply app/models/checkout.rb:1]: Let's not "resolve" the comments.  When an issue/bug is fixed just comment that it's fixed.  I like having the conversation out for notes when we look back at the code in the future.  It's good notes.

File: app/models/discover_flight.rb
Comment (timjedrek) [reply app/models/discover_flight.rb:1]: fixed

File: app/models/enrollment.rb
Comment (timjedrek) [reply app/models/enrollment.rb:1]: payload added

File: app/models/message.rb
[the remark this answers is not in the document]
Comment (timjedrek) [reply app/models/message.rb:1]: line 7

**Comment by timjedrek** [comment]
This is merged

**Pull request closed without merging**